cal/__init__.py

```diff
@@ -52,27 +54,21 @@ def page_not_found(e):
 
 @app.route('/')
 def home():
-    events = Event.query.order_by(Event.start).all()
-
-    sunday = [event for event in events if event.start.weekday() == 0]
-    monday = [event for event in events if event.start.weekday() == 1]
-    tuesday = [event for event in events if event.start.weekday() == 2]
-    wednesday = [event for event in events if event.start.weekday() == 3]
-    thursday = [event for event in events if event.start.weekday() == 4]
-    fridau = [event for event in events if event.start.weekday() == 5]
-    saturday = [event for event in events if event.start.weekday() == 6]
-
-    events = [sunday, monday, tuesday, wednesday, thursday, friday, saturday]
-    return render_template('index.html', events=events)
+    return render_template('index.html')
 
+@app.route("/events/")
+def events():
+    now = dt.datetime.now()
+    events = Event.query.filter(Event.start > now).\
```

Comment:
Should this be `Event.end`?
This query should be commented so we know what it's trying to retrieve.